src/services/account.js

```diff
@@ -0,0 +1,18 @@
+const fetchAccount = async (id, setAccount, setLoading, setError) => {
```

Comment:
Lets put this inside the existing services.accounts.js file. It shouldn't have a default export. 

Also, lets make use `function` vs the `() =>` fat arrow syntax since we do not need context in the stateless functions of the service.